docs/SPIDER_TYPE.md

```diff
@@ -0,0 +1,23 @@
+## Types of spiders
+
+The vast majority of spiders are collecting first party data about a specific brand/chain.
+We sometimes call these "brand spiders", these should be well categorised and branded by us, and have original data from
+the brand.
+We store them in [locations/spiders](../locations/spiders). and  they contain `spider:lineage=S_ATP_BRANDS` in the outputted
+`dataset_attributes`.
+
+We also collect government Open Data.
+These should also contain license details in `dataset_attributes`.
+These spiders are typically bigger than brand spiders, often collecting all of a specific category for a country,
+the age of the dataset, or records in the dataset, may be uncomfortably old.
+We store them in [locations/government_spiders](../locations/government_spiders), there lineage is `S_ATP_GOVERNMENTS`.
+
+We also collect infrastructure data, these wouldn't typically be called Points of Interest.
+These may be anything from trees to electrical sub stations.
```

Comment:
Some ideas for other examples to include: road and public transport features, council assets such as street trees, lamp posts, rubbish bins and park benches, utility assets such as manholes, electrical substations, pumping stations, etc.

Depending on where more useful POIs (such as playgrounds, public toilets, public showers near a beach, etc) end up, it might be worth also noting what should not be included under `infrastructure`.